client/packages/system/src/Patient/PatientView/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { LocaleKey } from '@common/intl';
+import { GenderType } from '@common/types';
+
+export function getGenderTranslationKey(gender: GenderType): LocaleKey {
```

Comment:
I _think_ this is the right way to go - our API returns a set of gender types, which we can map to translation keys.

However, the options available in the patient form are from the patient schema, which is probably custom per client. So they could have an entirely different set of display values for these gender options. But very hard to extract this out of UI schema - I think better to push the custom translations, get them to update their translations to match their schema at that point?

Reply:
Custom schemas should really have hard coded translations, only shareable schemas would need translations like this. Btw we do have [`t` now for translation of json forms](https://github.com/msupply-foundation/open-msupply/blob/0699638b856ea96dd33741aa549994c95450d9a9/standard_reports/stock-status/2_5/argument_schemas/arguments_ui.json#L39), this could also have been used in standard form (even for enums), I think I would prefer it done that way, unless we use all of the GenderType and translate them all. Also I think Hormone/Surgical shouldn't be translated (if no key exists)

Reply:
The problem I was meaning actually is the hard coded translations. They are available in the JSON form, but map to and enum type on base Patient which is queried by the toolbar here. 

Its really hard to pull the hard coded gender translations out of the UI schema (the gender picker may or may not exist, at any level of depth...) so hoping we are okay with OMS translations of gender enum in places where we arent rendering the form. 

I suppose other option would be to recommend always using translation strings even for custom gender options, once we get overrides in, so we can be sure they match?

Reply:
Ahh ok, that makes sense, I think what you have here is good cheers : )